pip_audit/_dependency_source/requirement.py

```diff
@@ -220,7 +219,7 @@ def _fix_file(self, filename: Path, fix_version: ResolvedFixVersion) -> None:
         #
         # This time we're using the `RequirementsFile.parse` API instead of `Requirements.from_file`
         # since we want to access each line sequentially in order to rewrite the file.
-        reqs = list(RequirementsFile.parse(filename=str(filename)))
+        reqs = list(RequirementsFile.parse(filename=filename.as_posix()))
```

Comment:
Confirming: is `as_posix` okay for Windows here?

Reply:
It's completely transparent. The `as_posix` only ensure that the path uses `/` instead of `\`.

Reply:
Okay. I couldn't remember if POSIX paths were fine on Windows or not, but if so this looks fine 🙂